From 80e1883dee774e4696dd8f88c8c86a513a618503 Mon Sep 17 00:00:00 2001
From: Neelansh Mittal <neelanshm@codeaurora.org>
Date: Sun, 29 Nov 2020 05:52:39 +0530
Subject: [PATCH 278/500] net: Add new xfrm_state xflag

New custom flag XFRM_STATE_OFFLOAD_NSS to identify
xfrm states offloaded to the NSS

Change-Id: Ib90a42fb9fd6818a6dfcb04e1ed35b4cfa6614c6
Signed-off-by: Neelansh Mittal <neelanshm@codeaurora.org>
Signed-off-by: Pavithra R <quic_pavir@quicinc.com>
---
 include/net/xfrm.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/net/xfrm.h b/include/net/xfrm.h
index 043b8fec76cf..f7e281d17543 100644
--- a/include/net/xfrm.h
+++ b/include/net/xfrm.h
@@ -299,6 +299,7 @@ static inline struct net *xs_net(struct xfrm_state *x)
 /* xflags - make enum if more show up */
 #define XFRM_TIME_DEFER	1
 #define XFRM_SOFT_EXPIRE 2
+#define XFRM_STATE_OFFLOAD_NSS 4
 
 enum {
 	XFRM_STATE_VOID,
-- 
2.34.1

